### ISSUE TITLE
 Pale Moon and Google Chrome problems

### ISSUE THREAD

**Issue by InstallGentoo11** [AHAAAAAAA/PokemonGo-Map#1998]
After the new UI was added the map stopped showing on Pale Moon. Any ideas what I have to enable or whatever to make it appear again? The option screen is there, but the map with everything on it is just gone.

On the other hand in Google Chrome there is some sort of blur on the both the map and the pokemon.

Both of the problems don't appear on Firefox or Chromium.

**Comment by chuyskywalker** [comment]
The blur, in my experience, is from having the page zoomed in.

**Comment by W1ndst0rm** [comment]
Have you tried clearing the cache? I had a similar issue to your top picture that was resolved that way. If you open the developer or error console, are there any errors?

**Comment by InstallGentoo11** [comment]
No, deleting cache doesn't help. And this is what I get from console.
![untitled](https://cloud.githubusercontent.com/assets/6108677/17109093/31858a12-529f-11e6-8e93-879e20a721b6.png)
